cypress/integration/grid_search.js

```diff
@@ -122,7 +122,7 @@ context("Grid Search", () => {
 		cy.get("@table")
 			.find('.grid-heading-row .search input[data-fieldtype="Duration"]')
 			.type("d");
-		cy.get("@table").find(".grid-body .rows .grid-row").should("have.length", 5);
+		cy.get("@table").find(".grid-body .rows .grid-row").should("have.length", 3);
 
```

Comment:
This is definitely correct. Rolling up 7 days to a week means there are less items with the duration having a “d” for days in the formatted string. That’s why either the assertion or the data or the duration options (e.g. “hide weeks”) needs to be adapted. The latter could be done if preferred, but this is fine, too.